elements/pl-file-editor/pl-file-editor.js

```diff
@@ -174,26 +207,50 @@ window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
       var fontSize = $(this).val();
       that.editor.setFontSize(fontSize);
     });
+    that.modal.find('#modal-' + uuid + '-keyboardHandler').change(function () {
+      var keyBinds = $(this).val();
+      if (keyBinds == 'Default') {
+        that.editor.setKeyboardHandler(null);
+      } else {
+        that.editor.setKeyboardHandler(keyBinds);
+      }
+    });
   });
 
   this.saveSettingsButton.click(function () {
     var theme = that.modal.find('#modal-' + uuid + '-themes').val();
     var fontsize = that.modal.find('#modal-' + uuid + '-fontsize').val();
+    var keybindings = that.modal.find('#modal-' + uuid + '-keyboardHandler').val();
```

Comment:
I believe you need the same logic to map `"Default"` to `null` that you have elsewhere.

Reply:
> I tested this out locally and it's mostly working great - it's so nice to have the Vim keybindings I know and love!
> 
> There's still some general confusion happening with the default keybindings. You can see this for yourself by opening your browser devtools and then switching around between bindings, it tries to load ones that don't exist:
> 
> <img alt="Screenshot 2022-11-30 at 15 04 12" width="1050" src="https://user-images.githubusercontent.com/1476544/204927008-b9a765ac-954c-4ef2-850b-ad29492058b5.png">
> 
> Note that `null` isn't a valid value for local storage, it's probably being converted to the string `"null"`. I'd recommend that you use `"Default"` when persisting to local storage and map that back to `null` when configuring the editor.

The getKeyHandler method might be the cause since I cannot find the ace editor's document. I instead use the value that is stored inside the local storage as my "currentKeyHandler value", and use this value in the session storage.

Reply:
Should be resolved now